src/python/pants/backend/codegen/thrift/java/java_thrift_library.py

```diff
@@ -39,6 +41,11 @@ def __init__(self,
       in the global options under ``--thrift-default-rpc-style``.
     :param namespace_map: An optional dictionary of namespaces to remap {old: new}
     :param thrift_linter_strict: If True, fail if thrift linter produces any warnings.
+    :param default_java_namespace: The namespace used for Java generated code when a Java
+      namespace is not explicitly specified in the IDL. The default is defined in the global
+      options under ``--thrift-default-default-java-namespace``.
+    :param include_paths: The include_path(s) used to search for included IDL files. The default
+      is defined in the global options under ``--thrift-default-include-paths``.
```

Comment:
IMO, let's not document this one. And if I can convince you to remove this parameter entirely, that would be preferable.

I understand that you want to use it for testing, but AFAICT, it should be possible to define dependencies that result in any interesting set of include-paths. As you saw above, include-paths are already passed to scrooge, and I imagine that if you stare at how they are generated for a few minutes, it will be clear how to cause other paths to be searched.